src/platform/backends/shared/linux/backend_utils.cpp

```diff
@@ -48,11 +56,22 @@
 
 namespace mp = multipass;
 namespace mpl = multipass::logging;
+namespace mpdbus = multipass::backend::dbus;
+
+typedef QMap<QString, QVariantMap> VariantMapMap;
+Q_DECLARE_METATYPE(VariantMapMap) // for DBus
 
 namespace
 {
 std::default_random_engine gen;
 std::uniform_int_distribution<int> dist{0, 255};
+const auto nm_bus_name = QStringLiteral("org.freedesktop.NetworkManager");
+const auto nm_root_obj = QStringLiteral("/org/freedesktop/NetworkManager");
+const auto nm_root_ifc = QStringLiteral("org.freedesktop.NetworkManager");
+const auto nm_settings_obj = QStringLiteral("/org/freedesktop/NetworkManager/Settings");
+const auto nm_settings_ifc = QStringLiteral("org.freedesktop.NetworkManager.Settings");
+const auto nm_connection_ifc = QStringLiteral("org.freedesktop.NetworkManager.Settings.Connection");
```

Comment:
We would. I thought of that, but the thing is, these really are independent names. You could have "a.b.c" bus name, "/d/e/f" object, and "g.h.i" interface AFAICT. They end up very similar like this by convention, which just tells me that DBus didn't solve namespacing very well :shrug: But let me know if you think it can be improved somehow.